docs/data/material/customization/color/ColorDemo.js

```diff
@@ -58,7 +58,7 @@ function ColorDemo(props) {
             </Typography>
           </Toolbar>
         </AppBar>
-        <Box component="pre" sx={{ m: 2, fontSize: 16 }}>
+        <Box component="pre" sx={{ m: 2, fontSize: 16, overflowX: 'auto' }}>
```

Comment:
Shouldn't it be

```suggestion
        <Box component="pre" sx={{ m: 2, fontSize: 16, overflow: 'auto' }}>
```

like in https://github.com/mui/material-ui/blob/494c10c6ff92b3072ca35564f7f24f05758e62e7/packages/mui-docs/src/MarkdownElement/MarkdownElement.tsx#L32?

Reply:
We could do this. The reason I went for `overflowX` is that I haven't come across any situation that requires a scrollbar on the Y axis.

Reply:
Ok, we can be lazy and wait. It's more that I usually never seen X/Y overflow to be superior to the alternatives: what I found most useful is nothing, or `overflow: auto` when I want to make sure a scroll container trigger.